src/main/java/baritone/cache/CachedWorld.java

```diff
@@ -106,10 +106,10 @@ public final boolean isCached(int blockX, int blockZ) {
     }
 
     @Override
-    public final LinkedList<BlockPos> getLocationsOf(String block, int maximum, int maxRegionDistanceSq) {
+    public final LinkedList<BlockPos> getLocationsOf(String block, int maximum, int centerX, int centerZ, int maxRegionDistanceSq) {
         LinkedList<BlockPos> res = new LinkedList<>();
-        int playerRegionX = playerFeet().getX() >> 9;
-        int playerRegionZ = playerFeet().getZ() >> 9;
+        int centerRegionX = centerX >> 9;
+        int centerRegionZ = centerZ >> 9;
```

Comment:
Can you remove the `implements Helper` for all the classes that no longer use it? (like this one I think)

Reply:
``CachedWorld`` actually does use it in other places, but removed it in all the other places where it's not used.